tests/services/ENSService.test.ts

```diff
@@ -9,8 +9,92 @@ describe('ENSService', () => {
       expect(ensService.isSupported('abc.eth')).toBe(true)
     })
 
-    it('do not support abc.bit', () => {
-      expect(ensService.isSupported('abc.bit')).toBe(false)
+    it('do not support abc.bit', async () => {
+      await expect(ensService.isSupported('abc.bit')).toBe(false)
+    })
+
+    it('leon.eth has been registered', async () => {
+      const isRegistered = await ensService.isRegistered('leon.eth')
+      expect(isRegistered).toBe(true)
+    })
+
+    it('leon.eth is available', async () => {
+      const isAvailable = await ensService.isAvailable('leon.eth')
+      expect(isAvailable).toBe(false)
+    })
+
+    it('address that owner of leon.eth is valid', async () => {
+      const owner = await ensService.owner('leon.eth')
+      expect(owner).toMatch(/^0x[0-9A-Fa-f]{40}$/)
+    })
+
+    it('tokenId that owner of leon.eth is valid', async () => {
+      const tokenId = await ensService.tokenId('leon.eth')
+      expect(tokenId).toMatch(/^0x[0-9A-Fa-f]{64}$/)
```

Comment:
`TokenId` of a specific name if fixed. So the test result should be a plain string.